phpBB/includes/compatibility_globals.php

```diff
@@ -44,7 +44,6 @@
 // Grab global variables, re-cache if necessary
 /* @var $config phpbb\config\db */
 $config = $phpbb_container->get('config');
-set_config(null, null, null, $config);
```

Comment:
Since it's a fallback method now, I guess we could also use a global instead of keeping the call. :speak_no_evil: